app/models/user.php

```diff
@@ -78,39 +105,92 @@ public static function get()
         return $row;
     }
 
+    /* CHECKERS FOR REGISTRATION */
+
+    //checks if firstname and lastname contain letters only
     public function nameChecker($name)
```

Comment:
Na-move na po yung nameChecker but the passwordCheck cannot be moved kasi po nagkakaerror na po.